src/Exception/InvalidArgumentException.php

```diff
@@ -1,14 +0,0 @@
-<?php
-
-namespace Http\Client\Exception;
-
-use Http\Client\Exception;
-
-/**
- * Custom Invalid Argument exception
- *
- * @author Márk Sági-Kazár <mark.sagikazar@gmail.com>
- */
-final class InvalidArgumentException extends \InvalidArgumentException implements Exception
```

Comment:
sorry, did not read well enough. yeah lets do that. can you link the place where you add the BatchResult in the utilities repo?